<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisOptionsDialog</class>
 <widget class="QDialog" name="AxisOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Axis Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox:drop-down {
border-image: url(:/core/drop-down-sm-blue.png);
border: 0px transparent;
}

QComboBox {
min-height: 33px;
min-width: 100px;
}

#AxisOptionsDialog {
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(208, 208, 208, 255));
}


SlideToggle {
min-width: 66px;
max-width: 66px;
min-height: 34px;
max-height: 34px;
}

SlideToggle::groove:horizontal {
 border-image: url(:/core/toggle-bg.png); 
border: 0px transparent;
 border-left: -9px transparent;
 border-right: -8px transparent;
 border-bottom: -5px transparent;
 border-top: 0px transparent;
/* border: 0px transparent;*/
 padding: 0px;
 margin: 0px;
}

SlideToggle::handle:horizontal {
margin:0px;
padding:0px;
image: url(:/core/slider-handle-blue.png);
}

SlideToggle {
margin:4px;
}

#masterFrame {
/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(172, 172, 172, 255), stop:1 rgba(146, 146, 146, 255));*/
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(172, 172, 172, 255), stop:1 rgba(146, 146, 146, 255));
border: 1px solid rgb(0,0,0);
border-radius: 6px;
max-height: 40px;
padding: 5px;
}

#masterFrame QLabel {
color: rgb(255,255,255);
font: 12pt &quot;Lucida Sans&quot;;
}

#buttonFrame {
border: 0px;
}

#buttonFrame QPushButton {
padding:10px;
color: rgb(255,255,255);
font: 12px &quot;Lucida Sans&quot;;
max-width:75px;
max-height: 24px;
}

#cancelButton {
border-image: url(:/core/small-gray-but-normal.png);
border: 0px transparent;
}

#saveButton {
border-image: url(:/core/small-blue-but-normal.png);
border: 0px transparent;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="defLabel">
     <property name="text">
      <string>Load Defaults</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="defaultCombo"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Axis Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeCombo"/>
   </item>
   <item>
    <widget class="QFrame" name="masterFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="masterLabel">
        <property name="text">
         <string>Master</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="SlideToggle" name="masterToggle">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ratioLabel">
     <property name="text">
      <string>Ratio</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ratioLine"/>
   </item>
   <item>
    <widget class="QLabel" name="maxLabel">
     <property name="text">
      <string>Max Steps/Sec</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxLine"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlideToggle</class>
   <extends>QSlider</extends>
   <header>core/Widgets/slidetoggle.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
